src/kaocha/plugin/junit_xml.clj

```diff
@@ -151,18 +173,25 @@
                 "Save the test results to a Ant JUnit XML file."
 
                 "--junit-xml-omit-system-out"
-                "Do not add captured output to junit.xml"]))
+                "Do not add captured output to junit.xml"
+
+                "--junit-xml-add-location-metadata"
+                "Add line, column, and file attributes to tests in junit.xml"]))
```

Comment:
Interestingly, when running `bin/kaocha --help`, I don't see these available, and they are considered errors by the CLI:
<img width="814" alt="image" src="https://user-images.githubusercontent.com/10106320/166892133-5bb27216-0073-4745-91d4-c07c17e8c5aa.png">